ValveResourceFormat/IO/ContentFormats/ValveMap.cs

```diff
@@ -167,7 +169,13 @@ internal class CMapSelectionSet : DMElement
 {
     public Datamodel.ElementArray Children { get; } = [];
     public string SelectionSetName { get; set; } = string.Empty;
-    public DMElement SelectionSetData { get; set; }
+    public CObjectSelectionSetDataElement SelectionSetData { get; set; } = [];
+}
+
+[CamelCaseProperties]
+internal class CObjectSelectionSetDataElement : DMElement
+{
+    public Datamodel.ElementArray selectedObjects { get; } = [];
```

Comment:
```suggestion
    public Datamodel.ElementArray SelectedObjects { get; } = [];
```
Since it's using camel case properties attribute.